src/pip/_internal/index/package_finder.py

```diff
@@ -745,7 +745,7 @@ def _log_skipped_link(self, link, reason):
             # the reason contains non-ascii characters.
             #   Also, put the link at the end so the reason is more visible
             # and because the link string is usually very long.
-            logger.debug(u'Skipping link: %s: %s', reason, link)
+            logger.debug('Skipping link: %s: %s', reason, link)
```

Comment:
The comment above about UnicodeEncodeError can be removed as well.

Reply:
Fixed in latest revision.